src/mobile/src/ui/components/CustomTextInput.js

```diff
@@ -226,10 +226,11 @@ class CustomTextInput extends Component {
     onChangeText(value) {
         const { isPasswordInput, isSeedInput, onValidTextChange } = this.props;
         if (isSeedInput) {
-            if (value && !value.match(VALID_SEED_REGEX)) {
+            const valueCapitalized = value.toUpperCase()
+            if (valueCapitalized && !valueCapitalized.match(VALID_SEED_REGEX)) {
```

Comment:
Both before this change and after -- so if `valueCapitalized` is `""`, the regex-check won't trigger, and the empty seed will go to `onValidTextChange` -- ?

Reply:
Should probably be
```javascript
if (!valueCapitalized || !valueCapitalized.match(VALID_SEED_REGEX)) {
    return;
}
```